UnityClient/Assets/Scripts/TowerDefence/Wave.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[System.Serializable]
+public class Wave
+{
+    public GameObject enemy;
+    public int count;
+    public float rate;
```

Comment:
public fields are a cardinal sin and go against the convention of the codebase. If you need Unity inspector support, please convert this to a `SerializeFieldAttribute` private field, as per the other places in the codebase. If you need access to this value outside of this type, please use a property instead.

If it does become private, please follow the codebase's private field convention and define the name `_inThisFormat`, as per other places in the codebase.